docs/configuration/prometheus-frontend.md

```diff
@@ -40,16 +40,19 @@ query_range:
         size: 1024
         validity: 24h
 
-      # If you want to use a memcached cluster, configure a headless service
+      # If you want to use a memcached cluster, you can either configure a headless service
       # in Kubernetes and Cortex will discover the individual instances using
-      # a SRV DNS query.  Cortex will then do client-side hashing to spread
-      # the load evenly.
+      # a SRV DNS query (host) or you can also list comma separated memcached addresses 
+      # of the cluster nodes in case it is not running within kubernetes 
+      # (EKS Elasticache/Managed memcached cluster). Cortex will then do client-side hashing
+      # to spread the load evenly. 
       # memcached:
-      #   memcached_client:
-      #     host: memcached.default.svc.cluster.local
-      #     service: memcached
-      #     consistent_hash: true
-
+      #   expiration : 24h
+      # memcached_client:
+      #   host: memcached.default.svc.cluster.local
+      #   addresses: ""
```

Comment:
Let me explain the current situation. You can configure the endpoint in two ways:
- `host` + `service`: this is the config you should set when you use the SRV DNS (this is considered stable)
- `addresses`: this is experimental and supports service discovery (https://cortexmetrics.io/docs/configuration/arguments/#dns-service-discovery) so it could either be a list of single addresses, or a SRV record prefixed with `dnssrvnoa+`

Could you clarify it accordingly, please?

Reply:
done, looks better ?